src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -115,6 +109,25 @@ public UpdateTranscriptionResponse updateTranscription(Integer transcriptionId,
         return updateTranscriptionResponse;
     }
 
+    private void validateUpdateTranscription(Integer currentTranscriptionStatusId,
+                                             UpdateTranscription updateTranscription) {
+
+        Map<TranscriptionStatusEnum, Set<TranscriptionStatusEnum>> expectedStatuses = new HashMap<>();
+        expectedStatuses.put(AWAITING_AUTHORISATION, Set.of(APPROVED, REJECTED));
```

Comment:
Can you remove the status from here and the tests as this is covered in my ticket

Reply:
I don't think I should take this basic validation out in this PR as Richard M. in the Test team has familiarised himself with the updateTranscription endpoint testing for Approved flow yesterday.
So it would be a regression if I removed it from this pull request - it would be better if your ticket ensured this functionality still worked with regard to the status validation for Approved / Rejected flows?
I thought you would be swapping this bit out just.  That way, I'd expect the error tests I've added to continue passing.

Reply:
My changes require it to send 409 Conflict response not bad request so I will have to rip out all your verification stuff.